core/templates/app.constants.ts

```diff
@@ -16,6 +16,11 @@
  * @fileoverview Shared constants for the Oppia module.
  */
 
+export interface WARNING_TYPES_CONSTANT {
+  CRITICAL: string,
+  ERROR: string
+}
```

Comment:
Hmm, can!t this be declared more simply? Like all values will be strings?

Reply:
@vojtechjelinek Yup it can be done by doing something like
```typescript
export interface WARNING_TYPES_CONSTANT {
  [errorType: string]: string
}
```
Should I change the definition to this?

Reply:
I think that we should go with this more generic version, @ankita240796 do you agree?

Reply:
Hmm, I think we should not go with the generic version. The concern I have is that the with a more generic, we allow addition of any string to the warning interface. We should try to make the types as strict as possible to avoid any wrong changes. What do you think @vojtechjelinek?

Reply:
Ok, makes sense.